pandas/tests/dtypes/test_common.py

```diff
@@ -204,6 +204,8 @@ def test_is_timedelta64_dtype():
     assert com.is_timedelta64_dtype(np.timedelta64)
     assert com.is_timedelta64_dtype(pd.Series([], dtype="timedelta64[ns]"))
 
+    assert not com.is_timedelta64_dtype("0 days 00:00:00")
```

Comment:
didn't you have 2 cases that trigger the exception? if so pls add the other one.